postgres/src/main/scala/zio/sql/postgresql/PostgresSqlModule.scala

```diff
@@ -308,6 +308,7 @@ trait PostgresSqlModule extends Sql { self =>
     val BitLength                   = FunctionDef[String, Int](FunctionName("bit_length"))
     val Pi                          = Expr.FunctionCall0[Double](FunctionDef[Any, Double](FunctionName("pi")))
     val GenRandomUuid               = Expr.FunctionCall0[UUID](FunctionDef[Any, UUID](FunctionName("gen_random_uuid")))
+    val DateTrunc                   = FunctionDef[(String, Instant), LocalDateTime](FunctionName("date_trunc"))
```

Comment:
Have you considered using the `ZonedDateTime`?
Looking at spec, this function supports the timezone.
https://www.postgresql.org/docs/current/functions-datetime.html#FUNCTIONS-DATETIME-TRUNC

Reply:
No problem using ZonedDateTime. I could refactor the code to use ZonedDateTime.

Reply:
Probably, I would go for `OffsetDateTime` but for consistency I also used the "zoned" one when implementing the `Now()` function.

@jczuchnowski @sviezypan I looked at existing implementation, and other methods are around ZonedDateTime. However, I'm not sure whether, in the DB context, it's not better to use OffsetDateTime. That's my experience from different projects, and also [here is an article](https://www.baeldung.com/java-zoneddatetime-offsetdatetime#differences) that also confirms that the OffsetDateTime will be what people will use in a DB related data model.

If you see it as reasonable, this might be a topic for a separate issue to align all functions.

Reply:
OffsetDateTime does look like a better solution.